catkin_ws/src/image_rotation/src/image_algorithm.cpp

```diff
@@ -0,0 +1,23 @@
+///
+/// @file
+/// @brief Implementation of image processing algorithm
+///
+
+#include "image_algorithm.h"
+
+using namespace cv;
```

Comment:
@melisamariam 
According to [Namespaces in Google Style guide C++](https://google.github.io/styleguide/cppguide.html#Namespaces), it is forbidden to use "using namespace foo;"
@upengareri : please make the necessary changes